src/pke/include/schemebase/rlwe-cryptoparameters.h

```diff
@@ -348,7 +388,10 @@ class CryptoParametersRLWE : public CryptoParametersBase<Element> {
     ar(::cereal::make_nvp("rw", m_relinWindow));
     ar(::cereal::make_nvp("md", m_maxDepth));
     ar(::cereal::make_nvp("mo", m_mode));
+    ar(::cereal::make_nvp("pmo", m_premode));
     ar(::cereal::make_nvp("slv", m_stdLevel));
+    ar(::cereal::make_nvp("fdp", m_floodingdistributionParameter));
+    m_dggflooding.SetStd(m_floodingdistributionParameter);
```

Comment:
move line 384 "m_dgg.SetStd(m_distributionParameter);" to the end of this function as well.

Reply:
fixed

Reply:
don't mark it fixed unless it is in the branch